controller/users.go

```diff
@@ -361,9 +381,55 @@ func (c *UsersController) Update(ctx *app.UpdateUsersContext) error {
 			}
 		}
 	}
+	c.notifyWITService(ctx, ctx.RequestData) // TODO: Dont ignore error
 	return returnResponse
 }
 
+// addDelegationFlag adds information for WIT service to know that
+// this was forwarded from AUTH
+func addDelegationFlag(ctx *app.UpdateUsersContext) *app.UpdateUsersContext {
+	ctx.Context = context.WithValue(ctx.Context, delegationFlag, true)
+	return ctx
+}
+
+func (c *UsersController) notifyWITService(ctx *app.UpdateUsersContext, request *goa.RequestData) error {
+	remoteWITClient, err := c.createRemoteWITClient(ctx.Context, request)
+	if err != nil {
+		return err
+	}
+	updateUserPayload := &witservice.UpdateUsersPayload{
+		Data: &witservice.UpdateUserData{
+			Attributes: &witservice.UpdateIdentityDataAttributes{
+				Bio:                   ctx.Payload.Data.Attributes.Bio,
+				Company:               ctx.Payload.Data.Attributes.Company,
+				ContextInformation:    ctx.Payload.Data.Attributes.ContextInformation,
+				Email:                 ctx.Payload.Data.Attributes.Email,
+				FullName:              ctx.Payload.Data.Attributes.FullName,
+				ImageURL:              ctx.Payload.Data.Attributes.ImageURL,
+				RegistrationCompleted: ctx.Payload.Data.Attributes.RegistrationCompleted,
+				URL:      ctx.Payload.Data.Attributes.URL,
+				Username: ctx.Payload.Data.Attributes.Username,
+			},
+			Type: ctx.Payload.Data.Type,
+		},
+	}
+	ctx = addDelegationFlag(ctx)
```

Comment:
We don't need to add any special flags.
Use the auth service account token instead (see my comments below).